Yafc/Windows/DependencyExplorer.cs

```diff
@@ -147,12 +150,14 @@ public class DependencyExplorer : PseudoScreen {
                         gui.BuildText("Status: Marked as inaccessible");
                         if (gui.BuildLink("Clear mark")) {
                             SetFlag(ProjectPerItemFlags.MarkedInaccessible, false);
+                            NeverEnoughItemsPanel.Refresh();
                         }
                     }
                     else {
                         gui.BuildText("Status: Not accessible. Wrong?");
                         if (gui.BuildLink("Manually mark as accessible")) {
                             SetFlag(ProjectPerItemFlags.MarkedAccessible, true);
+                            NeverEnoughItemsPanel.Refresh();
                         }
                     }
                 }
```

Comment:
Do I understand it correctly that you did not put `NeverEnoughItemsPanel.Refresh()` after `if (current.IsAccessible()){} else {}` because the terminal branches are hit rarely?

Reply:
Yes, if I understand you correctly. `if (current.IsAccessible()` happens every time something the UI might want to respond to happens, and I wanted hold off on the recalculation until something had actually changed.